src/components/editor/editor.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "./editor.module.scss";
+import Container from "../container/container";
+import Button from "../UI/button/button";
+
+export default function Editor() {
+  return (
+    <Container title="Share your thoughts" size="s">
+      <div className={styles.editor}>
+        <textarea
+          className={styles.textarea}
+          rows={4}
+          placeholder="Enter your message..."
+        ></textarea>
```

Comment:
лучше зашорткатить тег)
читаемей будет
```suggestion
        />
```